lesson_9/mosut_sorin/hw_lesson9_classes_mosut_sorin.py

```diff
@@ -0,0 +1,149 @@
+
+
+'''
+Build two classes of your choice that can model a real-life example.
+The class needs to meet the following requirements:
+
+    at least 5 attributes each
+
+    at least 2 methods each
+
+    one class to inherit from another
+
+As a demonstration create at least 5 instances of
+one class (preferably the child class) and
+call all the methods it holds
+
+Ex: You can have one class (Country) that has general attributes
+about countries such as area, neighbours, cities etc and
+methods related to those attributes. The second class can be a
+specific country (Romania) that has more specific attributes
+such as attractions, universities etc.
+'''
+
+# main class (parent class)
+class Automobil(object):
+    clasa = 'automobil'
+
+    def __init__(self, marca, tip, varianta, an, cilindree, putere):
+        self.marca = marca
+        self.tip = tip
+        self.var = varianta
+        self.an = an
+        self.cil = int(cilindree)
+        self.putere = int(putere)
+
+# method that converts the power from [kW] in [hp] and compare it wit a specific value
+    def test_putere(self):
```

Comment:
the name of the method can describe better the functionality and the return value.

`def print_message_medium_high_power(self)`

To explain the thought process: 
1. the method is only printing on screen -> print_message 
2. the if condition is about the power of the car -> _medium_high_power